projects/robots/abb/irb/controllers/inverse_kinematics/inverse_kinematics.py

```diff
@@ -80,7 +80,7 @@
 
 # Get the arm and target nodes.
 target = supervisor.getFromDef('TARGET')
-arm = supervisor.getSelf()
+arm = supervisor.getFromDef('ARM')
```

Comment:
It is safer to keep the `getSelf` as in any case in the world the `ARM` DEF is not defined.

Reply:
Done!